ihatemoney/templates/history.html

```diff
@@ -0,0 +1,235 @@
+{% extends "sidebar_table_layout.html" %}
+
+{% macro change_to_logging_preference(event) %}
+{% if event.val_after == LoggingMode.DISABLED %}
+    {% if event.val_before == LoggingMode.ENABLED %}
+        {{ _("Disabled Project History") }}
+    {% elif event.val_before == LoggingMode.RECORD_IP %}
+        {{ _("Disabled Project History & IP Address Recording") }}
+    {% else %}
+        {{ _("Disabled Project History & IP Address Recording") }}
+    {% endif %}
```

Comment:
The `elif` and `else` are identical. Is there any valid reason?

Reply:
Nope. I was just trying to make sure I didn't forget any cases. Fixed as of dc9a420b